modules/340-extended-monitoring/templates/x509-certificate-exporter/deployment.yaml

```diff
@@ -1,6 +1,6 @@
-{{- define "cert_exporter_resources" }}
+{{- define "x509_certificate_exporter_resources" }}
 cpu: 50m
-memory: 50Mi
+memory: 150Mi
```

Comment:
Max allowed memory resources is 100 Mi, min cannot be greater than max.

Reply:
Fixed